### PR TITLE
API Information Page PR Improvements

### PR DESCRIPTION
**Pull request by notatiyyah**
Improvements on #8 

- Make tests more robust by adding tests for multiple screen sizes & failing tests
- Improve documentation (readme, adding the right website logo and name, etc..)
- Fix issue where API information pages are not accessible directly (must be redirected from a link on the API Catalogue page)
- Fix linting errors & CSS interference errors causing unintended side effects

### CODE REVIEW

**Review by 1JW1 — APPROVED** [review]
This looks good, I particularly liked the range of the cypress tests and the different devices it takes into account

**Review by humulla — COMMENTED** [review]
A couple of suggestions, Also, based on the last PR I don't see any failing scenario tests in the API information page, Are you still working on this?

File: cypress/integration/previewApi.spec.js [hunk 1]
```diff
@@ -14,22 +14,26 @@ describe("Preview an API", () => {
 
     screenSizes.forEach((screenSize) => {
         it(`View API name on ${screenSize} screen`, function () {
+            cy.viewport(screenSize);
```
[the remark this answers is not in the document]
Comment (notatiyyah) [reply cypress/integration/previewApi.spec.js:17]: Yes, I realised there was a bug where the API Info was not visible on mobile, but no tests failed. I added some tests to make sure everything is still visible on all screen sizes :)

**Comment by notatiyyah** [comment]
> Based on the last PR I don't see any failing scenario tests in the API information page, Are you still working on this?

@humulla
https://github.com/LBHackney-IT/developers-hub-frontend/blob/53bf6c8df0a56784673ca976f30908896ee506a5/cypress/integration/viewApiInformation.spec.js#L62-L72

**Review by humulla — APPROVED** [review]

**Pull request merged**